rotkehlchen/db/evmtx.py

```diff
@@ -287,7 +287,7 @@ def get_transaction_hashes_not_decoded(
     def count_hashes_not_decoded(
             self,
             chain_id: Optional[ChainID],
-            addresses: Optional[list[ChecksumEvmAddress]],
```

Comment:
why make it optional? Explicit `=None` is cleaner code. Generaly avoid default args if possible.